src/DataStructures.jl

```diff
@@ -39,12 +42,14 @@ module DataStructures
     export SortedDict, SortedMultiDict, SortedSet
     export SDToken, SDSemiToken, SMDToken, SMDSemiToken
     export SetToken, SetSemiToken
-    export startof
+    export startof, endof ## both are deprecated
```

Comment:
We don't need to explictly export deprecated things `@deprecate` does that for us

```suggestion
```